package.json

```diff
@@ -8,8 +8,7 @@
     "url": "https://github.com/Microsoft/azure-devops-extension-sdk.git"
   },
   "engines": {
-    "node": ">=18.0.0",
-    "npm": ">=10.0.0"
```

Comment:
I removed the npm requirement because this refers to the npm version people need to _install_ this package, not _publish_. And most people don't have npm 10 yet because that was only released in an LTS Node version yesterday.